packages/web/localizations/en.json

```diff
@@ -13,7 +13,7 @@
     "scalingFactorControllerWarning_third": "to maintain the peg. The current scaling factor is",
     "scalingFactorInformation": "This factor allows the curve to adjust to predictable price changes, such as the value accruing to a liquid staking token.",
     "singleAsset": "Single asset",
-    "stablecoinWarning": "You are adding liquidity to {denom}, a {mechanism}-backed stablecoin.",
+    "stablecoinWarning": "You are adding liquidity to {denom}, a {mechanism} stablecoin.",
```

Comment:
Thoughts on the fact that the types are not localized? We could have a different key for each type so that they get translated.

Also, we'll need to localize the remaining files. Please take a look at the readme in the root of the repo for how this is done.

Reply:
different keys for each type are localized now with most of the translations. just trying to find any help with cn, hk, tw languages.